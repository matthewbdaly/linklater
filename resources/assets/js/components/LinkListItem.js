// @flow
import React from 'react';

type Props = {
  link: string,
  title: string
};

const LinkListItem = (props: Props) => {
  return (
    <li className="list-group-item">
      <a href={ props.link } target="_blank" rel="noopener noreferrer">{ props.title }</a>
    </li>
  );
};

export default LinkListItem;
