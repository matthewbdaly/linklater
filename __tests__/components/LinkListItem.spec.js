import React from 'react';
import {findDOMNode} from 'react-dom';
import LinkListItem from '../../resources/assets/js/components/LinkListItem';
import renderer from 'react-test-renderer';

describe('LinkListItem', () => {
  it('renders the link list item', () => {
    const tree = renderer
      .create(<LinkListItem link='' title='' />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
