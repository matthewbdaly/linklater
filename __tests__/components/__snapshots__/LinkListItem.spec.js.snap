// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LinkListItem renders the link list item 1`] = `
<li
  className="list-group-item"
>
  <a
    href=""
    target="_blank"
  >
    
  </a>
</li>
`;
